core/lib/spree/testing_support/factories/payment_factory.rb

```diff
@@ -6,6 +6,10 @@
     order
     state 'checkout'
     response_code '12345'
+
+    factory :payment_completed do
+      state 'completed'
+    end
```

Comment:
We may not be using them elsewhere, but this might be a good use for traits... 

`trait :completed do
   complete true
end`
`FactoryGirl.create(:payment, :completed)`